swift/ql/src/queries/Security/ECB-Encryption/ECBEncryption.ql

```diff
@@ -0,0 +1,81 @@
+/**
+ * @name Encryption using ECB
+ * @description Highlights uses of the encryption mode 'ECB'. This mode should normally not be used because it is vulnerable to replay attacks.
+ * @kind path-problem
+ * @problem.severity warning
+ * @security-severity 7.5
```

Comment:
Note: This number is produced by a [tool](https://github.com/github/cwe-scores) we have in a private repo. Have you used that tool to generate this score? Or is this a hand-picked number?

Reply:
There are expected security-severity values in the query priorities table, which I'm guessing is where this number comes from.  They're sourced from queries with similar tags as a guide to what we might expect, they may well be accurate, but we shouldn't assume them to be correct.

Reply:
That number is the same as a [similar query](https://github.com/github/codeql/blob/main/csharp/ql/src/Security%20Features/Encryption%20using%20ECB.ql) for C#.

Reply:
- I'll arrange for the DCA run with you offline then.
- I don't seem to have enough permission to add labels to the PR though.

Reply:
The script gives me the same result as you have already:
```
 * @security-severity 7.5
```
🎉